packages/wrangler/src/cli.ts

```diff
@@ -1,5 +1,35 @@
+import * as Sentry from "@sentry/node";
+import process from "process";
 import { hideBin } from "yargs/helpers";
+import { RewriteFrames } from "@sentry/integrations";
+
 import { main } from ".";
+import * as pkj from "../package.json";
+
+Sentry.init({
+  release: `${pkj.name}@0.0.11`,
+  initialScope: {
+    tags: { [pkj.name]: pkj.version },
+  },
+  dsn: "https://87383452498e420ebdb01de6100e44ec@o226679.ingest.sentry.io/6139281",
```

Comment:
Should this URL be stored in the code for public consumption? This could allow people to send fake data to our logging or try to overload it...

If we got esbuild to read it from the environment at build time then we could hide it in our CI.

Reply:
They'd still be able to see it in the bundle. This is fine really. 
What we do need before we land this PR is to figure out an opt-out system.

Reply:
> They'd still be able to see it in the bundle

Doh! Of course. But it would be nice not to have this constant hidden inside the code here, IMO. Perhaps not a build time ENV but at least pulled out into a `const` somewhere obvious.

Reply:
Regarding opt-out. The Angular CLI uses a postinstall hook: https://github.com/angular/angular-cli/blob/b3d7080147f755d578d2ee5611437aed9c88f098/packages/angular/cli/bin/postinstall/analytics-prompt.js#L15

Reply:
It then has two levels of config: project (local) and global. For the project level, the equivalent would be to store it in the wrangler.toml. For the global then we would could define a file like `~/.wrangler.toml`.